RussianFilter/sections/specific.txt

```diff
@@ -88,6 +88,7 @@ hirologos.com##.uk-container > #tm-bottom-b
 eg.ru,kino-teatr.ru,ren.tv##a[href="http://mirtesen.ru/"]
 aif.ru##.related_content_block
 bin.ua##div[style*="width:240px; height:400px;"]
+/engine/modules/banners/wot*$image,domain=wotspeak.org
```

Comment:
`/engine/modules/banners/` is not popular. I prefer to use `||wotspeak.org/engine/modules/banners/` It seems it contains only ads.